lib/matplotlib/tests/test_datetime.py

```diff
@@ -755,8 +793,32 @@ def test_triplot(self):
     @pytest.mark.xfail(reason="Test for violin not written yet")
     @mpl.style.context("default")
     def test_violin(self):
-        fig, ax = plt.subplots()
-        ax.violin(...)
+        plt, ax = plt.subplots(1, 1)
```

Comment:
```suggestion
        fig, ax = plt.subplots(1, 1)
```

This is flagging `flake8` because it is reusing the name for `pyplot`, the idiomatic name for this variable is `fig`, as it is a `Figure` object.